src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTest.cs

```diff
@@ -1475,6 +1475,11 @@ public async Task SendAsync_Unexpected1xxResponses_DropAllInterimResponses(HttpS
             {
                 await server.AcceptConnectionAsync(async connection =>
                 {
+                    if (connection is Http2LoopbackConnection http2Connection)
```

Comment:
`SetupAutomaticPingResponse()` initializes an out-of-band automatic response to any PING received by the `LoopbackServer` forcing to ignore PING whenever a test does `ReadFrameAsync()`. Enabling it by default seems also beneficial to me, but it will also immediately change the semantics of **all** tests in the test suite, so I was hesitant to do so.

If you are absolutely sure that this is the right thing to do, I will change the behavior and refactor all the tests, removing explicit PING initialization.

Reply:
At the very least it should be automatic for any test using GenericLoopbackServer/LoopbackServerFactory. These tests are version agnostic and therefore should not need to have HTTP2-specific logic.

For HTTP2-specific tests, I don't care as much.